jupyterlite_xeus/__init__.py

```diff
@@ -11,4 +11,4 @@
 
 
 def _jupyter_labextension_paths():
-    return [{"src": "labextension", "dest": "@jupyterlite/xeus"}]
+    return [{"src": "labextension", "dest": "@jupyterlite/xeus"}, {"src": "liteextension", "dest": "@jupyterlite/xeus-kernels-extension"}]
```

Comment:
Hopefully we won't need the `liteextension` split soon with https://github.com/jupyterlite/jupyterlite/pull/1590.